src/components/user-profile.html

```diff
@@ -1,125 +1,142 @@
         width: 400px;
+        padding-left: 10px;
+        display: contents;
+      }
+
+      td {
+        padding-left: 20px;
+        font-size: 18px;
+        vertical-align: bottom;
       }
+
       button {
-        background-color: DarkSlateBlue;
+        background-color: indigo;
         color: #fcfcff;
         border-radius: 3px;
         text-align: center;
         padding: 12px;
         margin-top: 20px;
         font-size: 12px;
+        margin-top: 30px;
+        margin-left: 10px;
+        margin-right: 10px;
+        width: 100px;
+      }
+
+      button:hover {
+        opacity: 0.9;
+        transition: all 0.3s ease;
       }
+
       label {
-        margin-top: 25px;
         padding-right: 5px;
+        margin-top: 15px;
+        padding-top: 10px;
       }
+
       input {
         font-size: 18px;
         border: 0;
         border-bottom: 1px solid indigo;
         background: transparent;
-      }  
-      .add-button {
-        background-color: #fca902;
-        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-        padding: 20px;
-        margin-top: 0px;
-        font-size: 18px;
-        padding-right: 40px;
-        padding-left: 40px;
+        margin-top: 5px;
+        padding-top: 10px;
       }
-      template {
-        display: flex;
-        justify-content: center;
+
+      .delete {
+        background-color: #d1250ee3;
       }
     </style>
     <template is="dom-if" if="[[addUserSelected]]">
       <table>
         <tr>
-          <td><label for="firstName">First name :</label></td>
+          <td><label for="firstName">First name </label></td>
           <td><input type="text" name="firstName" id="firstName" value="{{user.firstName}}" on-input="updateUserFirstName"></td>
         </tr>
         <tr>
-          <td><label for="lastName">Last name :</label></td>
+          <td><label for="lastName">Last name </label></td>
           <td><input type="text" name="lastName" id="lastName" value="{{user.lastName}}" on-input="updateUserLastName"></td>
         </tr>
         <tr>
-          <td><label for="phone">Phone :</label></td>
+          <td><label for="phone">Phone </label></td>
           <td><input type="text" name="phone" id="phone" value="{{user.phoneNumber}}" on-input="updateUserPhoneNumber"></td>
         </tr>
         <tr>
-          <td><label for="email">Email :</label></td>
+          <td><label for="email">Email </label></td>
           <td><input type="text" name="email" id="email" value="{{user.email}}" on-input="updateUserEmail"></td>
         </tr>
-        <tr>
-          <td><button type="button" on-click="newProfile">[[changeButtonText(addUserSelected)]]</button></td>
-        </tr>
       </table>
+      <button type="button" on-click="newProfile">[[changeButtonText(addUserSelected)]] <jha-icon-save style="height: 16px; width: 16px; vertical-align: text-bottom; padding-left: 5px; fill:#fcfcff;"></jha-icon-save></button>
```

Comment:
you should be able to reference `jha-icon-save` in your style tag and put all this inlined stuff up there.

Reply:
I tried that before, that way is way more ideal but this is all I could get to work. It says the only way to personally style it is in-line in the guide. I might try something different later.